README.md

```diff
@@ -125,7 +125,7 @@ Example request URI:
 | ```INTERNAL_SERVER_ERROR(500)```        | ```Internal ID could not be retrieved from Auth```
 
 
-#### POST /register  
+#### POST /register-ordinary-partnership 
```

Comment:
I think the route should be register-general-partnership as this is the more widely used term

Reply:
You should also update the readme for the Register Scottish Partnership API as well